### ISSUE TITLE
dot files(for e.g.babelrc) does not get copied to app dir, useful with Next 2.0 beta release.

### ISSUE THREAD

**Issue by aga5tya** [mars/heroku-nextjs-build#2]
@mars, First off thanks for the wonderful helper script. 

In my current app which runs on nextjs 2.0.0-beta.17, we use .babelrc file to override default transforms for e.g. to use with [styled-jsx-postcss](https://github.com/giuseppeg/styled-jsx-postcss). I see at present the dot files are ignored by default, i tried to modify the script to allow .dotfiles with the below

```mv $BUILD_DIR/* $BUILD_DIR/.[!.]* $APP_DIR/```

but end up with the error below. Could you please point me the right direction.

```
mv: inter-device move failed: ‘/app/nextjs/.’ to ‘/tmp/build_aa4a104bff15ad63fed8f1066e0f5f11/nextjs/.’; u
nable to remove target: Is a directory
mv: inter-device move failed: ‘/app/nextjs/..’ to ‘/tmp/build_aa4a104bff15ad63fed8f1066e0f5f11/nextjs/..’;
 unable to remove target: Is a directory
```

**Comment by mars** [comment]
Thanks for reporting this issue @aga5tya ,

I just released a fix to keep .dotfiles. It's now available on npm as [version 1.1.0](https://www.npmjs.com/package/@mars/heroku-nextjs-build).

**Comment by aga5tya** [comment]
:clap: Thank you @mars
